boot/makebootable.go

```diff
@@ -407,6 +406,12 @@ func makeRunnableSystem(model *asserts.Model, bootWith *BootableSet, sealer *Tru
 		Grade:          string(model.Grade()),
 		ModelSignKeyID: model.SignKeyID(),
 	}
+	// XXX: ideally this would use boot.Participant() but that
+	//  requires quite a bit of refactor because we need to pass a
+	//  snap.Device instead of a model here
```

Comment:
I would drop the XXX, the change would be a a bit of a +-0. I would add a comment restating that on classic there is no boot base though.